src/app/content/components/Page/searchHighlightManager.ts

```diff
@@ -77,6 +79,11 @@ const searchHighlightManager = (container: HTMLElement) => {
   };
 
   return {
+    scrollTo: (searchHighlight: Highlight) => () => {
+      allImagesLoaded(services.container).then(
```

Comment:
this `allImagesLoaded` should be done in general before triggering a scroll from the scrollTargetManager, not specifically for search highlights.